### ISSUE TITLE
osmWebWizard cannot open the web

### ISSUE THREAD

**Issue by MollyIKnight** [eclipse/sumo#8711]
Hello,

I run the linux build in wsl v1 ubuntu 20.04, but it never opens the openstreetmap webpage in the browser, and no message is given in the termial. I am not sure if it's because of my settings in the wsl. I have set the path of browser to chrome and I can open website with ```wslview``` cmd in chrome.

If I ctrl+c in the terminal I get:

Traceback (most recent call last):
  File "tools/osmWebWizard.py", line 546, in <module>
    server.serveforever()
  File "/mnt/d/programming/sumowin/tools/webWizard/SimpleWebSocketServer.py", line 625, in serveforever
    rList, wList, xList = select(

**Comment by namdre** [comment]
try opening  /mnt/d/programming/sumowin//tools/webWizard/index.html in your browser directly

**Comment by MollyIKnight** [comment]
ah yes, seems to work (run osmWebWizard.py in wsl, open index.html in browser at the same time). Thanks.

**Comment by MollyIKnight** [comment]
This issue happens again... and I cannot save the map data to the net file this time.

Powershell gives this error.

Traceback (most recent call last):
  File "D:\programming\sumo\tools\osmWebWizard.py", line 544, in <module>
    server = SimpleWebSocketServer(
  File "D:\programming\sumo\tools\webWizard\SimpleWebSocketServer.py", line 595, in __init__
    self.serversocket.bind((host, port))
OSError: [WinError 10013] An attempt was made to access a socket in a way forbidden by its access permissions

**Comment by MollyIKnight** [comment]
I downloaded map directly from osm and used netconvert to generate a net.xml. It works, but I would like to set up the osmWebWizard correctly for future use ...

**Comment by namdre** [comment]
There are many potential causes of the reported error message, most having to do with local configuration setting (https://stackoverflow.com/questions/10461257/an-attempt-was-made-to-access-a-socket-in-a-way-forbbiden-by-its-access-permissi)
- firewall rules
- port already in use
- windows policy for local connections

You could try to work down the list of potential remedies from stackoveflow.

**Comment by behrisch** [comment]
please reopen if problems persist and you think we can do something on our side.

**Comment by MollyIKnight** [comment]
Thanks for the help. I was working on sth else so didn't get back to this sooner ... Will try the stackoverflow suggestions later.

**Comment by weishanlee** [comment]
> wsl
Could you explain what wsl is?

**Comment by MollyIKnight** [comment]
wsl stands for windows subsystem linux. It is enabled in windows system settings, then you can install a linux app to get a linux subsystem in windows system.

I don't think I get the importing map issue solved with the stackflow suggestions. In the end I built the simulation config files in the windows exe version of SUMO, then copied the files to the linux version of SUMO and ran in wsl ubuntu, so that I can specify sampling parameters. There probably are easier ways to do this.

**Comment by behrisch** [comment]
I did never use the WSL with GUI applications which seems to be a quite recent feature and it may not be well supported yet. What hinders you to do everything directly in Windows?

**Comment by weishanlee** [comment]
Currently, I cannot open the website by clicking on the osmWebWizard icon. However, I can open the website by running the osmWebWizard.py with the python IDLE. I do not know why, but it works for me. I already installed the traci package for python.

**Comment by behrisch** [comment]
Please open a new ticket and describe in detail especially which python, windows and sumo version you are using. Thanks!

**Comment by MollyIKnight** [comment]
I can't remember very well what cmd I used for simulation now, it's been a year. As far as I can remember, I used linux build for specifying parameters for fcd output, such as simulation time, sampling rate, sampling precision, sampling time period. I didn't read the win documentation very well so I didn't know if win build offered setting simulation parameters in fcd output cmd too. I just needed the work done by the deadline, to be honest, so I went on with using linux build after I got the map data sorted with win build.

I currently don't need support on this issue any more. If someone else has the same or similar issue, they can open a new ticket with their detailed description. I might not be able to find the detailed cmds or the same environment I used back then, after the system and python updates I got over the year ....
